### ISSUE TITLE
Build fails in Unity 2018.1

### ISSUE THREAD

**Issue by unnamed7** [resonance-audio/resonance-audio-unity-sdk#24]
This is same issue as GVR's [https://github.com/googlevr/gvr-unity-sdk/issues/861](url)

ResonanceAudioBuildProcessor needs to implement the new interface otherwise the build fails.

Workaround is same as above add this

```
#if UNITY_2018_1_OR_NEWER
  public void OnPreprocessBuild(BuildReport report) {
    OnPreprocessBuild(report.summary.platform, report.summary.outputPath);
  }
#endif
```

to the body of ResonanceAudioBuildProcessor.cs

Takashi

**Comment by anokta** [comment]
Thanks for reporting Takashi! Indeed, that's the fix - will be updated with the next release.

See resonance-audio/resonance-audio@e1923fe6fe733ae4d3c8460ff883c87e2ad05d6b for reference.

**Comment by anokta** [comment]
Fixed in [v1.2.1](https://github.com/resonance-audio/resonance-audio-unity-sdk/releases/tag/v1.2.1).
